### PR TITLE
Juno docs page

### PR DESCRIPTION
**Pull request by jessiemongeon1**
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

### CODE REVIEW

**Review by dfx-json — APPROVED** [review]

File: docs/developer-docs/web-apps/browser-js/juno.mdx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+Juno provides a way to deploy dapps onto ICP without using the traditional ICP development workflow that utilizes [`dfx`](/docs/current/developer-docs/getting-started/install/) to create, manage, and deploy canisters.
+
+## Using Juno
+
+To get started with Juno, you can create your first [satellite](https://juno.build/docs/add-juno-to-an-app/create-a-satellite), a 100% on-chain smart contract container that provides secure data storage, authentication, hosting, and also supports custom domains. Juno satellite deployment and management are done through a web-based [console](https://console.juno.build/).
+
+To integrate frameworks and packages with Juno, the [Juno SDK core library](https://www.npmjs.com/package/@junobuild/core) is used. This library can be downloaded using [npm](https://www.npmjs.com/), then used in your local environment to integrate Juno with frameworks such as Vue, Angular, Rust, or React.
```
Comment (peterpeterparker) [inline docs/developer-docs/web-apps/browser-js/juno.mdx:13]: To integrate frameworks and packages into your applications and start building, follow one of the quick-start [guides and tutorials](https://juno.build/docs/category/guides-and-examples). Juno is agnostic, allowing for seamless integration with your favorite frontend technologies, whether you're using Next.js, React, SvelteKit, Vue, Angular, Astro, or any other framework.

Furthermore, if you're aiming to extend the capabilities, you can enhance the features with serverless behaviors running within smart contracts using Rust.

File: docs/developer-docs/web-apps/frameworks/juno.mdx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Juno is a cutting-edge blockchain-as-a-service platform designed to enable developers to create decentralized applications at lightning speed. It can be thought of as an innovative, open-source alternative to Google Firebase, but with the added benefits of the Internet Computer technology.
+
+It features a growing list of plug-and-play services such as authentication, data management, file storage, hosting, serverless functions, and analytics. These tools empower developers to efficiently bring their innovative ideas to life.
+
+Juno provides a way to deploy dapps onto ICP without using the traditional ICP development workflow that utilizes [`dfx`](/docs/current/developer-docs/getting-started/install/) to create, manage, and deploy canisters.
+
+## Using Juno
+
+To get started, you can create your first [satellite](https://juno.build/docs/add-juno-to-an-app/create-a-satellite), a 100% on-chain smart contract container offers all of JUno's features. Juno satellite deployment and management are done through a web-based [console](https://console.juno.build/), a CLI, or GitHub Actions.
```
Comment (peterpeterparker) [inline docs/developer-docs/web-apps/frameworks/juno.mdx:13]: Small typo `JUno`

File: docs/developer-docs/web-apps/frameworks/juno.mdx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Juno
+
+## Overview
+
+Juno is a cutting-edge blockchain-as-a-service platform designed to enable developers to create decentralized applications at lightning speed. It can be thought of as an innovative, open-source alternative to Google Firebase, but with the added benefits of the Internet Computer technology.
+
+It features a growing list of plug-and-play services such as authentication, data management, file storage, hosting, serverless functions, and analytics. These tools empower developers to efficiently bring their innovative ideas to life.
+
+Juno provides a way to deploy dapps onto ICP without using the traditional ICP development workflow that utilizes [`dfx`](/docs/current/developer-docs/getting-started/install/) to create, manage, and deploy canisters.
```
Comment (peterpeterparker) [inline docs/developer-docs/web-apps/frameworks/juno.mdx:9]: As mentioned previously, I think this paragraph should be removed or the non-Juno related links should be removed, given that the goal is to maximize the chance for the reader to continue their journey with/to Juno and not with some other tools.

**Review by peterpeterparker — APPROVED** [review]
LGTM, thanks a lot!

P.S.: Before merge, spotted a small missing space in

"utilizes `dfx`to" 
-> 
"utilizes `dfx` to"

**Pull request merged**